docs/content/Getting-Started/quickstart-subs/quickstart-2-apache-example-deployment-a-kind-clusters-florin.md

```diff
@@ -14,4 +14,4 @@ EOF
 ```  
 
 Note: if you already have a cluster named 'florin' from a previous exercise of KubeStellar, please delete the florin cluster ('kind delete cluster --name florin') and create it using the instruction above.
-<!--quickstart-2-apache-example-deployment-a-kind-clusters-florin-end-->
\ No newline at end of file
+<!--quickstart-2-apache-example-deployment-a-kind-clusters-florin-end-->
```

Comment:
What is the change here?

Reply:
I only changed line number 5, I'm not sure why it's picking up line 17. A few but not all of the files I touched git says I changed the last line but shows no obvious difference. I made edits with vim, no clue...

Reply:
It seems the original files for which I "edited" the last line did not terminate with LF, so Vim added those in.

Reply:
In the case of adding a terminal linebreak, I know that I have seen GitHub call that out by showing a red circle-with-line to indicate lack of terminal linebreak.  Of course, there are two cases: adding and removing.  I wonder if that makes the difference.